draft-hodges-webauthn-registries.xml

```diff
@@ -235,146 +276,209 @@
+            <t>
+              Description: The "packed" attestation format
               is a WebAuthn-optimized format for attestation
               data. It uses a very compact but still extensible encoding method. This
               format is implementable by authenticators with limited resources (e.g.,
-              secure elements).</t>
-              <t>Specification Document: <xref target="WebAuthn"/></t>
-            </list>
-          </t>
-<t><vspace blankLines="1" /></t>
-          <t>
-            <list style="empty">
-              <t>WebAuthn Attestation Formats: tpm</t>
-              <t>Description: The TPM attestation format returns an attestation statement in the
+              secure elements).
+            </t>
+            <t>
+              Specification Document: <xref target="WebAuthn"/>
+            </t>
+          </list>
+        </t>
+        <t>
+          <vspace blankLines="1" />
+        </t>
+        <t>
+          <list style="empty">
+            <t>WebAuthn Attestation Formats: tpm</t>
+            <t>
+              Description: The TPM attestation format returns an attestation statement in the
               same format as the packed attestation format, although the the rawData and signature
-              fields are computed differently.</t>
-              <t>Specification Document: <xref target="WebAuthn"/></t>
-            </list>
-          </t>
-<t><vspace blankLines="1" /></t>
-          <t>
-            <list style="empty">
-              <t>WebAuthn Attestation Formats: android</t>
-              <t>Description: Android-based, platform-provided authenticators may produce an attestation statement based on the Android SafetyNet API.</t>
-              <t>Specification Document: <xref target="WebAuthn"/></t>
-            </list>
-          </t>
-<t><vspace blankLines="1" /></t>
-          <t>
-            <list style="empty">
-              <t>WebAuthn Attestation Formats: android2</t>
-              <t>Description: Platform-provided authenticators based on Android versions "N", and
-              later, may provide a "hardware attestation" statement.</t>
-              <t>Specification Document: <xref target="AndroidHWAttstn"/></t>
-            </list>
-          </t>
-<t><vspace blankLines="1" /></t>
-          </section> <!-- sctn-attstn-format-reg-cnts -->      
-      
-      
-        <section title="Initial WebAuthn Extension Identifiers Registry Contents" anchor="sctn-extn-id-reg-cnts">
-          <t>The WebAuthn Extension Identifiers registry's initial contents are:</t>
+              fields are computed differently.
+            </t>
+            <t>
+              Specification Document: <xref target="WebAuthn"/>
+            </t>
+          </list>
+        </t>
+        <t>
+          <vspace blankLines="1" />
+        </t>
+        <t>
+          <list style="empty">
+            <t>WebAuthn Attestation Formats: goog-android</t>

```

Comment:
(b) says you suggest a vendor prefix, but doesn't supply justification -- what might that be?